### PR TITLE
Add plugin_test

### PR DESCRIPTION
**Pull request by nzws**
https://github.com/asdf-community/.github/issues/2

### CODE REVIEW

**Comment by imbsky** [comment]
Oops, and could you add this to plugin_test job?

```yml
env:
  GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -33,6 +33,19 @@ jobs:
...
+        os: [ubuntu-latest, macOS-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: asdf_plugin_test
+        uses: asdf-vm/actions/plugin-test@v1.0.0
+        with:
+          command: ocaml --version
+
```
Comment (imbsky) [inline .github/workflows/workflow.yml]:
```suggestion
        env:
          GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

Comment (nzws) [reply .github/workflows/workflow.yml]: Thank you!🙇

**Comment by imbsky** [comment]
Thanks @nzws for your contribution!

**Comment by imbsky** [comment]
@all-contributors please add @nzws for infra

**Pull request merged**